README.md

```diff
@@ -1,22 +1,31 @@
-[Azure Image Builder](https://docs.microsoft.com/azure/virtual-machines/image-builder-overview) (AIB) is a managed service that generates virtual machine (VM) images for you, and is built upon [HashiCorp Packer](https://www.packer.io). We'll use this service to build a general-purpose AKS jumpbox image that you could consider using as a starting point for your own jumpbox image. While this image could be built in other ways, Azure Image Builder helps customers think about their jumpbox image in a way that can be successfully managed via infrastructure-as-code and integrated into build pipelines.
+[Azure Image Builder](https://docs.microsoft.com/azure/virtual-machines/image-builder-overview) (AIB) is a managed service that generates virtual machine (VM) images for you, and is built upon [HashiCorp Packer](https://www.packer.io). We'll use this service to build a general-purpose AKS jump box image that you could consider using as a starting point for your own jump box image. While this image could be built in other ways, Azure Image Builder helps customers think about their jump box image in a way that can be successfully managed via infrastructure-as-code and integrated into build pipelines.
 
 ![An image showing a VM template snippet moving through the Azure Image Builder service, resulting in a Managed VM Image.](./image-builder-process.png)
 
 > The image generated by this specific process has not undergone any specific hardening or integration with security agents (antivirus, FIM, etc). Before you take a dependency on this, or any jumpbox image, ensure it complies with your requirements.
 
-While these instructions are targeting the building of a general-purpose AKS jumpbox VM image, these instructions could be referenced for other VM image purposes (such as build agents). These instructions do NOT address pushing images to a Shared Image Gallery, using Gen-2 VMs, or any other features of the Azure Image Builder service. To see the full features of the service, [review the service's documentation](https://docs.microsoft.com/azure/virtual-machines/image-builder-overview).
+While these instructions are targeting the building of a general-purpose AKS jump box VM image, these instructions could be referenced for other VM image purposes (such as build agents). These instructions do NOT address pushing images to a Shared Image Gallery, using Gen-2 VMs, or any other features of the Azure Image Builder service. To see the full features of the service, [review the service's documentation](https://docs.microsoft.com/azure/virtual-machines/image-builder-overview).
+
+## Installed software
+
+* [Azure CLI](https://docs.microsoft.com/cli/azure/what-is-azure-cli)
+* [kubectl](https://kubernetes.io/docs/reference/kubectl/overview/) and [kubelogin](https://github.com/Azure/kubelogin)
+* [Helm CLI](https://helm.sh)
+* [Flux v2 CLI](https://fluxcd.io)
+* [Open Service Mesh CLI](https://openservicemesh.io)
+* HashiCorp [Terraform CLI](https://www.terraform.io)
```

Comment:
what is terraform for?

Reply:
Nothing in our reference implementation, but this is a general purpose jump box, so I plan on adding common tooling such as this.  Akin to how general purpose build agents have tooling like TF installed as well.